runelite-client/src/main/java/net/runelite/client/plugins/objectindicators/ObjectIndicatorsOverlay.java

```diff
@@ -95,15 +100,15 @@ else if (object instanceof GroundObject)
 
 			if (polygon != null)
 			{
-				OverlayUtil.renderPolygon(graphics, polygon, config.markerColor());
+				OverlayUtil.renderPolygon(graphics, polygon, color);
 			}
 
 			if (polygon2 != null)
 			{
-				OverlayUtil.renderPolygon(graphics, polygon2, config.markerColor());
+				OverlayUtil.renderPolygon(graphics, polygon2, color);
 			}
-		}
 
+		}
```

Comment:
Remove the newline above this line, and re-add the one below it. (it looks strange on the diff)